matlab/test/Ice/slicing/objects/AllTests.m

```diff
@@ -484,9 +481,13 @@
             b2 = b1.pb;
             assert(~isempty(b2));
             assert(strcmp(b2.sb, 'D3.sb'));
-            assert(strcmp(b2.ice_id(), '::Test::B'));  % Sliced by server
-            assert(b2.pb == b1);
-            assert(~isa(b2, 'Test.D3'));
+            if proxy.ice_getEncodingVersion() == Ice.EncodingVersion(1, 0)
+                assert(~isa(b2, 'Test.D3'));
+            else
+                assert(isa(b2, 'Test.D3'));
+                assert(strcmp(b2.ice_id(), '::Test::D3'));  % preserved
+                assert(b2.pb == b1);
```

Comment:
Should we put this assert outside the else block? `pb` is not Sliced for 1.0 encoding as it is part of the base class B.

Reply:
Fixed